testsuite/do_tests.sh.in

```diff
@@ -732,11 +735,17 @@ if test "x${TEST_PYNEST}" = xtrue ; then
         @PYTHON@ @NOSETESTS@ -v --with-xunit --xunit-file="${TEST_OUTDIR}/pynest_tests.xml" "${NEST_PYTHON_PREFIX:-@CMAKE_INSTALL_PREFIX@/@PYEXECDIR@}/nest/tests" 2>&1 \
             | tee -a "${TEST_LOGFILE}" | grep -i --line-buffered "\.\.\. ok\|fail\|skip\|error" | sed 's/^/  /'
 
-        PYNEST_TEST_TOTAL="$(  tail -n 3 ${TEST_LOGFILE} | grep Ran | cut -d' ' -f2 )"
-        PYNEST_TEST_SKIPPED="$(  tail -n 1 ${TEST_LOGFILE} | sed -$EXTENDED_REGEX_PARAM 's/.*SKIP=([0-9]+).*/\1/')"
-        PYNEST_TEST_FAILURES="$( tail -n 3 ${TEST_LOGFILE} | grep \( | sed -$EXTENDED_REGEX_PARAM 's/^[a-zA-Z]+ \((.*)\)/\1/' | sed -$EXTENDED_REGEX_PARAM 's/.*failures=([0-9]+).*/\1/' )"
-        PYNEST_TEST_ERRORS="$( tail -n 3 ${TEST_LOGFILE} | grep \( | sed -$EXTENDED_REGEX_PARAM 's/^[a-zA-Z]+ \((.*)\)/\1/' | sed -$EXTENDED_REGEX_PARAM 's/.*errors=([0-9]+).*/\1/' )"
+        ntsummary=`@PYTHON@ -c "import junitparser as jp; r = jp.JUnitXml.fromfile('${TEST_OUTDIR}/pynest_tests.xml'); pl = ['.'.join((s.classname, s.name)) for s in r if s.result and not isinstance(s.result, jp.junitparser.Skipped)]; print(r.tests, r.skipped, r.failures, r.errors, pl)"`
```

Comment:
I would prefer if the executed code were put in its own script and given the `TEST_OUTDIR` as an argument. Also, I'd find it nicer if the execution of the subshell were done using the `$( ... )` syntax intsead of <code>\`...\`</code>, as this is also used (almost) everywhere else in the script.